server/backend-api-system/src/main/scala/cool/graph/system/mutations/PushMutation.scala

```diff
@@ -51,7 +51,7 @@ case class PushMutation(
         errors = List(
           SchemaError(
             "Global",
-            "Only projects that have been ejected can make use of the CLI's deploy function. More details: https://docs-next.graph.cool/reference/service-definition/legacy-console-projects-aemieb1aev"
+            "Only projects that have been ejected can make use of the CLI's deploy function. More details: https://www.graph.cool/docs/reference/service-definition/legacy-console-projects-aemieb1aev/"
```

Comment:
Don't update the link to the old docs. The correct link is: https://docs-next.graph.cool/docs/reference/service-definition/legacy-console-projects-aemieb1aev

Reply:
Never mind, just noticed that those are actually the new docs...